README.md

```diff
@@ -1,3 +1,112 @@
-# tdr-terraform-environments
+# TDR Terraform Environments
 
-Readme for tdr-terraform-environments repository
+This respository contains the Terraform code to create the AWS resources needed to support the TDR application.
+
+## Terraform Structure
+
+The prototype is divided into separate Terraform modules that represent the different AWS resources that are needed to for the TDR project.
+
+The different modules are used by Terraform workspaces which represent three AWS environments:
+
+* intg
+* staging
+* prod
+
+## Getting Started
+
+### Terraform Backend
+
+Ensure that the Terraform backend has been created.
+
+See here: https://github.com/nationalarchives/tdr-dev-documentation/blob/tkAWSInfrastructureSetup/tdr-create-aws-instructure-setup.md
+
+### Install Terraform locally
+
+See: https://learn.hashicorp.com/terraform/getting-started/install.html
+
+### Install AWS CLI Locally
+
+See: https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html
+
+### Install Terraform Plugins on Intellij
+
+HCL Language Support: https://plugins.jetbrains.com/plugin/7808-hashicorp-terraform--hcl-language-support
+
+### Add AWS Credentials and Profiles
+
+1. Update local AWS credentials file (~/.aws/credentials) with Terraform user credentials for the TDR AWS management account:
+
+   ```
+   ... other credentials ...
+
+   [terraform]
+   aws_access_key_id = ... terraform user access key ...
+   aws_secret_access_key = ... terraform user secret access key ...
+   ```
+   
+2. Update local AWS configuration file (~/.aws/config) with the profiles for running Terraform in the different TDR environments:
+
+   ```
+   ... other profiles ....
+   
+   [profile intgterraform]
+   region = eu-west-2
+   role_arn = ... terraform role arn for intg environment ...
+   source_profile = terraform
+   
+   [profile stagingterraform]
+   region = eu-west-2
+   role_arn = ... terraform role arn for staging environment ...
+   source_profile = terraform
+   
+   [profile prodterraform]
+   region = eu-west-2
+   role_arn = ... terraform role arn for prod environment ...
+   source_profile = terraform   
+   ```
+   Depending the Terraform workspace selected Terraform will use the relevant profile to update the TDR environment that corresponds to the Terraform workspace.   
```

Comment:
Should "Depending the Terraform workspace selected" be "Depending _on_ the Terraform workspace selected"?

Actually, you could drop that whole phrase - the rest of the sentence still makes sense without it.

Reply:
resolved